backend/tournesol/tests/test_api_criteria_correlation.py

```diff
@@ -0,0 +1,110 @@
+
+import numpy as np
+from django.test import TestCase
+from rest_framework import status
+from rest_framework.test import APIClient
+
+from core.tests.factories.user import UserFactory
+from tournesol.models.ratings import ContributorRatingCriteriaScore
+from tournesol.tests.factories.entity import EntityFactory
+from tournesol.tests.factories.poll import (
+    CriteriaFactory,
+    CriteriaRankFactory,
+    PollFactory,
+    PollWithCriteriasFactory,
+)
+from tournesol.tests.factories.ratings import (
+    ContributorRatingCriteriaScoreFactory,
+    ContributorRatingFactory,
+)
+from tournesol.views.criteria_correlations import compute_correlation, compute_slope
+
+
+def test_compute_correlation():
```

Comment:
`test_compute_correlation` and `test_compute_slope` are outside the test case for a reason?

Reply:
They don't use the setup and don't touch the database so it felt natural to keep them out of the test case class.